### ISSUE TITLE
Google Charts - multiple charts on same canvas?

### ISSUE THREAD

**Issue by sanket360** [google/google-visualization-issues#2293]
Does Google supports multiple charts like Bar, Line and Bubble chart on same canvas (similar to any trading website eg. Google Finance or Yahoo Finance)?

**Comment by dlaliberte** [comment]
You can draw multiple charts on the same page, if that's what you mean by
'canvas'.   See this page for some things you need to watch out for:
https://developers.google.com/chart/interactive/docs/basic_multiple_charts

There is also a 'combo' chart that lets you combine several varieties of
charts in the same chart:
https://developers.google.com/chart/interactive/docs/gallery/combochart

On Sun, Jun 12, 2016 at 5:43 AM, sanketkubal notifications@github.com
wrote:

> Does Google supports multiple charts like Bar, Line and Bubble chart on
> same canvas (similar to any trading website eg. Google Finance or Yahoo
> Finance)?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/google/google-visualization-issues/issues/2293, or mute
> the thread
> https://github.com/notifications/unsubscribe/AAizDVTMioT2uKCHs78WQ5SlrX2HYAlZks5qK9TPgaJpZM4IzwKB
> .

## 

Daniel LaLiberte https://plus.google.com/100631381223468223275?prsrc=2
dlaliberte@Google.com dlaliberte@google.com   5CC, Cambridge MA

**Comment by sanket360** [comment]
Yes @dlaliberte I am looking for combo chart. In your sample I can see 2 charts Bar and Line are combined. **Can I add another series of Bubble chart** along with these bars and line?

Also, how to do i provide user feature to enabled / hide each series?

**Comment by nbering** [comment]
@sanketkubal A [DataView](https://developers.google.com/chart/interactive/docs/reference#dataview-class) can be used to enable/disable series without affecting the source data. You might also consider [Controls and Dashboards](https://developers.google.com/chart/interactive/docs/gallery/controls) if you are looking for some ui widgets and stuff to help you provide interactivity.

**Comment by dlaliberte** [comment]
The bubble chart features cannot be combined with other types of chart features yet.  You can get a similar effect, however, by using a 'scatter' series where you specify the size and color of each point with a 'style' role column.  I am working on an implementation of exactly that idea which will eventually be used as a replacement for the BubbleChart.

**Comment by dlaliberte** [comment]
We don't have any convenient feature to let users show or hide series, but you can implement something like that by handling clicks on the legend.

**Comment by sanket360** [comment]
Thanks @dlaliberte & @nbering for inputs.
